server/ec2spotmanager/views.py

```diff
@@ -378,20 +379,20 @@ def __handleConfigPOST(request, config):
 
 
 @deny_restricted_users
-def createConfig(request):
+def createEc2Config(request):
     if request.method == 'POST':
-        config = PoolConfiguration()
-        return __handleConfigPOST(request, config)
+        config = Ec2PoolConfiguration()
+        return __handleEc2ConfigPOST(request, config)
     elif request.method == 'GET':
-        configurations = PoolConfiguration.objects.all()
+        configurations = Ec2PoolConfiguration.objects.all()
 
         if "clone" in request.GET:
-            config = get_object_or_404(PoolConfiguration, pk=int(request.GET["clone"]))
+            config = get_object_or_404(Ec2PoolConfiguration, pk=int(request.GET["clone"]))
```

Comment:
The type of the `config` object should depend on what the user is trying to create.